tools/hyphy/hyphy_meme.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0"?>
+<tool id="hyphy_gard" name="HyPhy-MEME" version="@VERSION@+galaxy0">
+    <description>: Recombination Detection</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <version_command>HYPHYMP --version | tail -n 1 | awk '{print $1}'</version_command>
+    <command detect_errors="exit_code">
+    <![CDATA[
+    export MPI_NODE_COUNT=\${GALAXY_SLOTS:-1} &&
+    ln -s '$input_file' meme_input.fa &&
+    ln -s '$input_nhx' meme_input.nhx &&
+    echo $gencodeid > tool_params &&
+    echo `pwd`/meme_input.fa >> tool_params &&
+    echo `pwd`/meme_input.nhx >> tool_params &&
+    echo "1" >> tool_params &&
+    echo "0.1" >> tool_params &&
```

Comment:
Shouldn't we allow these to be input by the user?

Reply:
They aren't on datamonkey.org, so I decided to keep that here.